lib/src/lifecycle_module.dart

```diff
@@ -964,8 +970,10 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
   /// A utility to logging LifecycleModule lifecycle events
   void _logLifecycleEvents(
       String logLabel, Stream<dynamic> lifecycleEventStream) {
-    listenToStream(lifecycleEventStream, (_) => _logger.finest(logLabel),
-        onError: (error) => _logger.warning('$logLabel error: $error'));
+    listenToStream(lifecycleEventStream,
+        (_) => _logger.finest(logLabel, null, StackTrace.current),
```

Comment:
Does we know whether this significantly affects performance? There are a lot of lifecycle events that get logged during app initialization.

Reply:
On warnings, which should happen much less frequently, I think it's fine; I'm just worried about normal logs

Reply:
Hmm, fair point. I probably got a little over-zealous with that one.